src/Sfa.Tl.ResultsAndCertification.Api.Client/Clients/DfeSignInApiClient.cs

```diff
@@ -39,27 +42,32 @@ public async Task<DfeUserInfo> GetDfeSignInUserInfo(string organisationId, strin
 
             var userInfoResult = userInfo.Result;
             var ukprn = organisationUkprn.Result;
```

Comment:
this variable is for an Organisation object. Not a UKPRN

Reply:
done the change